apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -484,7 +485,9 @@ export const ScreeningDecisionDialog = ({
             <>
               {intl.formatMessage(
                 poolSkill?.type?.value === PoolSkillType.Nonessential &&
-                  !experienceAttachedToSkill
+                  (!experienceAttachedToSkill ||
+                    poolSkill?.skill?.category.value ===
```

Comment:
this makes every behavioural skill is unclaimed even if it is connected to experience. But as per Desfam channel this is what they wanted seems like. Let's go with this. Thanks!